mobify/astro-tutorial
# Launch Image

Update launch image and manually hide when page is finished loading.

Linked PRs: https://github.com/mobify/astro-scaffold/pull/143

## Changes
- Manually dismiss launch image

## How to test-drive this PR
- Run tutorial, ensure launch image appears right away and eventually disappears

## TODOS:
~~- [ ] Update tutorial documentation in the `dev` folder (in the Astro repo) to match the modified tutorial.~~
~~- [ ] Preview the docs to make sure your changes look good (see [here](https://github.com/mobify/astro#documentation) for instructions on how to preview).~~
